### PR TITLE
Added required keys in generated schema

### LINKED ISSUE #16: requirement section missing
After generating a JSON, there are no required keys.

...
    "required": [
      "id",
      "name",
      "price",
      "dimensions",
      "warehouseLocation"
    ],
    "title": "Product"
  }
}

is missing

### PR DESCRIPTION
**Pull request by hisankaran**
Resolves #16

### CODE REVIEW

**Comment by Eomm** [comment]
I think you could update the Travis yaml, but this should drop node < 6 (that are in EOL state right now) or change the `includes` since it has been added in node 6

File: src/schemas/json.js [hunk 1]
```diff
@@ -214,6 +215,9 @@ module.exports = function Process (title, object) {
     processOutput = processObject(object)
     output.type = processOutput.type
     output.properties = processOutput.properties
+    output.required = Object.keys(object).filter(function (key) {
+      return !['$schema'].includes(key)
```
Comment (Eomm) [inline src/schemas/json.js]:
```suggestion
      return !key.startsWith('$')
```

**Comment by hisankaran** [comment]
Thanks for the suggestion @Eomm!

**Comment by sshim-cb** [comment]
Will this get merged any time soon?

**Comment by sshim-cb** [comment]
Any updates on this?

**Pull request merged**